airflow/contrib/hooks/gcp_dataflow_hook.py

```diff
@@ -21,9 +21,23 @@
 import warnings
 
 # pylint: disable=unused-import
-from airflow.gcp.hooks.dataflow import DEFAULT_DATAFLOW_LOCATION, DataFlowHook, DataflowJobStatus  # noqa
+from airflow.gcp.hooks.dataflow import DEFAULT_DATAFLOW_LOCATION, DataflowHook, DataflowJobStatus  # noqa
```

Comment:
Same here @michalslowikowski00. Can you check your other AIP-21 PR's as well, please :)

```suggestion
from airflow.gcp.hooks.dataflow import DEFAULT_DATAFLOW_LOCATION, DataflowHook, DataflowJobStatus
```

Reply:
Sure. :)